"""empty message

Revision ID: b8d53a427749
Revises: 165fdb190dd0
Create Date: 2024-04-24 10:00:33.892980

"""
from alembic import op
import sqlalchemy as sa
from sqlalchemy.exc import OperationalError


# revision identifiers, used by Alembic.
revision = 'b8d53a427749'
down_revision = '165fdb190dd0'
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    try:
        with op.batch_alter_table('case__connector__instance', schema=None) as batch_op:
            batch_op.add_column(sa.Column('identifier', sa.String(), nullable=True))
    except OperationalError:
        print("Column 'identifier' already exist in 'case__connector__instance'")

    try:
        with op.batch_alter_table('task__connector__instance', schema=None) as batch_op:
            batch_op.add_column(sa.Column('identifier', sa.String(), nullable=True))
    except OperationalError:
        print("Column 'identifier' already exist in 'task__connector__instance'")

    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('task__connector__instance', schema=None) as batch_op:
        batch_op.drop_column('identifier')

    with op.batch_alter_table('case__connector__instance', schema=None) as batch_op:
        batch_op.drop_column('identifier')

    # ### end Alembic commands ###
